<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenu</class>
 <widget class="QDialog" name="SettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Himalaya</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="accessibleName">
   <string>Настройки</string>
  </property>
  <widget class="QCheckBox" name="autostart_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Автозапуск программы</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveOut">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Himalaya</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveOut_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Himalaya</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
